apps/api/app/api/v1/endpoints/oauth.py

```
@@ -0,0 +1,412 @@
+        # Store user info in our database
+        await store_user_info(name, email, picture_url)
+
+        # Return token via deep link - desktop app will handle storage
+        token = auth_response.sealed_session or auth_response.access_token
+        return RedirectResponse(url=f"gaia://auth/callback?token={token}")
+
+    except HTTPException as e:
+        logger.error(f"HTTP error during WorkOS desktop auth: {e.detail}")
+        return RedirectResponse(url=f"gaia://auth/callback?error={e.detail}")
+
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)